src/Catalyst.TestUtils/FileSystemBasedTest.cs

```diff
@@ -48,9 +48,9 @@ namespace Catalyst.TestUtils
     public class FileSystemBasedTest : IDisposable
     {
         protected readonly string CurrentTestName;
-        protected IFileSystem FileSystem;
+        public IFileSystem FileSystem;
         protected readonly ITestOutputHelper Output;
-        private DirectoryInfo _testDirectory;
+        public DirectoryInfo TestDirectory;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)